docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
```

Comment:
I would rather start by defining simulation model. What is the simulation model? Then we could use "simulation model parameters"

Reply:
I do think this is clear (and we can expect some background knowledge).

Reply:
please remember that we have a very long DPPS glossary, which means there are a lot of terms which we think we should be used to. I think it is always a good idea to define the language we are speaking before speaking it. But ok if you prefer to leave this way then.